.config/theme_overrides/404.html

```diff
@@ -21,8 +21,9 @@
 
 <!-- Content -->
 {% block content %}
-<div class="leadtext">404</div>
-<h1>We looked, but couldn’t quite find that page.</h1>
+<h1>404</h1>
+<div class="leadtext">We looked, but couldn’t quite find that page.</div>
+
 <h2>
```

Comment:
@garaynakca I had to load it locally to see the changes:

![Screenshot of 404 page](https://user-images.githubusercontent.com/7096681/227343075-4a09ba21-36ed-4c11-a8c8-920c0ecd1f9f.png)

Can you change the `<h2>` to plain `<div>` as well? Also I think in another branch you are reducing the size of the leadtext?

Reply:
Ah I see it now. I merged it in.